src/librustdoc/html/static/main.js

```diff
@@ -129,6 +129,16 @@ function hideThemeButtonState() {
     themePicker.style.borderBottomLeftRadius = "3px";
 }
 
+function removeEmptyStringsFromArray(x) {
```

Comment:
That's great! Just one last thing: please move this function inside the anonymous one just below: it's not being used anywhere else so no need to keep it in the global scope.